subiquity/server/controllers/install.py

```diff
@@ -268,6 +268,8 @@ async def postinstall(self, *, context):
         packages = []
         if self.model.ssh.install_server:
             packages = ['openssh-server']
```

Comment:
Again, not part of the PR, but maybe we should packages.append() instead of assigning directly.

Reply:
yeah i thought that when writing it. will change.